src/gui/tracks/TrackContentWidget.cpp

```diff
@@ -360,6 +359,14 @@ bool TrackContentWidget::canPasteSelection( TimePos clipPos, const QMimeData* md
 	QDomElement clipParent = dataFile.content().firstChildElement("clips");
 	QDomNodeList clipNodes = clipParent.childNodes();
 
+	// If we are pasting into the PatternEditor, only a single Clip is allowed to be pasted
+	// so we don't have the unexpected behavior of pasting on different PatternTracks
+	if( m_trackView->trackContainerView()->fixedClips() == true &&
+			clipNodes.length() > 1 )
```

Comment:
I believe the comments about [coding style](https://github.com/LMMS/lmms/wiki/Coding-Conventions) on Discord is regarding the `if` statement. Style lords say **if (statement)**, so it's about where the spaces are placed. It should probably look something like **if (m_trackView-> ... > 1)**
I find it reasonable to adapt the code to the surrounding code though.

Reply:
Exactly. That's what i meant to say but don't know whether i missed anything else. Also, i think the curly braces should come on the same line as the statement (I remember seeing some code in that style). Am not sure tho.

Reply:
> Also, i think the curly braces should come on the same line as the statement

No. Link to coding style chapter above.

Reply:
> No. Link to coding style chapter above.

Checked it now to confirm. My earlier assumption was wrong. Anyways good to know.

Reply:
Got it, the original code was styled correctly (afair, back when I wrote it) but I think at some point during reorg it might have been automatically changed to this other code style and I thought it was a new guideline being used. I definitely prefer LMMS's coding style, those parenthesis look less readable IMO. Anyways, I'm changing the affected lines to adhere to the guidelines, and the rest of the file can be fixed in another PR to make reviewing easier.